packages/myst-common/src/extractParts.ts

```diff
@@ -9,10 +9,9 @@ import { copyNode } from './utils';
  */
 export function selectBlockParts(tree: Root, part: string) {
   const blockParts = selectAll('block', tree).filter((block) => {
-    if (!block.data?.tags && !block.data?.part) return false;
```

Comment:
I think that this is actually the only line of this PR that we want to retain, deprecating being able to control the part of the document using a tag. I think it easier to create a new PR and close this one @fwkoch.